### ISSUE TITLE
Refactoring

### ISSUE THREAD

**Issue by ghost** [Victori2a/4_Lab#1]
Reviewer contact: https://t.me/strok1n
1. .idea и Lab3.iml добавить в gitignore.
2. Убрать закомментированные строки кода (например, в Main.java).
3. В перечислении ItemType перемешано большое количество независимых друг от друга свойств. Это перечисление следует разделить на несколько перечислений, например: LightLevel, Size, и т.д.
4. Неясно зачем класс Fragment (который следует переименовать в BookFragment или BookExcerpt) наследует класс Book.
5. Убрать свойство "name : String" у всех классов, которые наследуют класс Item, так как в этом классе уже есть это свойство.
6. Перечисление BodyType следует разделить на два перечисления, которые назвать FaceType и HairType, ситуация такая же, как и с перечислением ItemType.
7. Все методы должны что-то делать с объектами, а не просто вызывать System.out.prinln (Такими плохими методами, например, являются Human.read, Human.say). 
8. What the hell is that? В Интерфейсе Position все методы default. Нужно сделать их не default, или сделать класс вместо интерфейса.
![image](https://github.com/Victori2a/4_Lab/assets/148381112/7e1ea48f-aeac-4383-bd25-c30ee9cd0a62)

**Comment by Hanqnero** [comment]
Не согласен с первым пунктом. Если клонировать репозиторий на другую машину, то IntelliJ **благодаря этим файлам** восстановить настройки проекта.

**Comment by ghost** [comment]
🤔 Maybe.
